commons/src/main/java/com/ge/predix/acs/utils/HttpServletRequestUtil.java

```diff
@@ -1,50 +0,0 @@
-/*******************************************************************************
```

Comment:
After speaking with frank the logic encapsulated in HttpServletRequestUtil was moved to tokenlib